portfolio/src/main/java/com/google/sps/servlets/FormServlet.java

```diff
@@ -75,10 +92,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     // Obtain a list of at most 'max' comments
-    Query query = new Query("Comment");
+    Query query = new Query("ImageComment");
```

Comment:
So this is creating a new entity namespace `ImageComment`, what happens to the existing `Comments` in DataStore, are they still there? Is it possible to augment existing entities and add new fields to them?

Ideally, once we've started building out the capstone project, we would be able to lock down the database schema, and only perform migrations on it instead of creating new entities each time.

Reply:
As I have everything set up now, I could have kept the name Comment and added the url field and all pre-existing Comment entities would have just not had anything in that field. I was having some issues, however, and wanted to walk through the function order & changed the name to make sure I was getting everywhere--I just manually deleted all the Comment entities via the backend such that Datastore now only stores ImageComments, but I can change it back if you prefer.

Reply:
It's fine now, but thinking towards the future. Consider if we have a database with 100,000 `Comment` entities containing users' data. I'm wondering how we can expand and augment the schema in place, without blowing away all of that user data.